docs/website/docs/user-guides/advanced/in-cluster-devfile-registry.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Using Devfile Registries declared in the cluster
+sidebar_position: 6
+tags: ["devfile-registry", "registry", "in-cluster"]
+slug: using-in-cluster-devfile-registry
+---
+
+Besides getting the list of Devfile Registries to work with from the [local configuration file](../../overview/configure.md#managing-devfile-registries),
+`odo` can automatically detect Devfile Registries declared in the current cluster, and use them.
+
+It does so by detecting the presence of the following [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in the current cluster:
+- `ClusterDevfileRegistriesList`: installable at the cluster level
+- `DevfileRegistriesList`: installable at the namespace level
+
+More details on the [Devfile Registry Operator documentation](https://github.com/devfile/registry-operator/blob/main/REGISTRIES_LISTS.md).
```

Comment:
You could add a note that such registries are not necessarily running into the cluster.

Reply:
Done in [`2fe2050` (#6689)](https://github.com/redhat-developer/odo/pull/6689/commits/2fe2050ae08b2febf9cc18cc0f885733fb8eb616)